app/controllers/api/v2/people_actions.rb

```diff
@@ -14,6 +14,10 @@ def create_and_render
       render_person(person, :created)
     end
 
+    def show_and_render
+      render_person(person, :ok)
```

Comment:
See comments further on but I think this is the point at which an explicit refresh of person details from Nomis should be triggered, rather than as a side effect of querying the persons `category`. It would be clearer to understand exactly what's happening and mean more idiomatic implementation of the associated serializer and model. Something like

```ruby
def show_and_render
  person.refresh_from_nomis!
  render_person(person, :ok)
end
```

Reply:
Discussed this offline, this is fine to leave as is.